resources/views/home/support-the-game.blade.php

```diff
@@ -136,7 +101,7 @@
                 <div class="supporter__block supporter__block--{{'bg-'.$index % 3}}">
                     @switch ($group['type'])
```

Comment:
@include("home._supporter_perk_".($group['view'] ?? snake_case($group['name'])), ['perks' => $group])

(or just set `view` for all of them)